### ISSUE TITLE
Props are black (rp_southside_day)

### ISSUE THREAD

**Issue by mcNuggets1** [Facepunch/garrysmod-issues#4916]
### Details
After reloading the map three times, some props turned black.
https://i.imgur.com/pUpm3T0.jpg

I fixed it by simply doing a r_flushlod.

### Steps to reproduce
It seems to be random or when too many props are loaded or the map is huge in size.

**Comment by Lexa013** [comment]
Contact the add on authors it’s not a garrysmod related issue

**Comment by FlorianLeChat** [comment]
No, there was a similar issue in the past with the [light cache](https://github.com/Facepunch/garrysmod-issues/issues/2815), it's probably related.
